metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -1288,10 +1289,16 @@ def process_dashboard(
                 dashboard_id=dashboard_id,
                 fields=fields,
             )
-        except SDKError:
+        except (SDKError, DeserializeError) as e:
+            logger.error(f"Failed to load dashboard from Looker API Error: {str(e)}")
             # A looker dashboard could be deleted in between the list and the get
+            error_type = (
+                "Deserialization Error"
+                if isinstance(e, DeserializeError)
+                else "SDK Error"
+            )
             self.reporter.report_warning(
-                title="Error Loading Dashboard",
+                title=f"Error Loading Dashboard: {error_type}",
                 message="Error occurred while attempting to loading dashboard from Looker API. Skipping.",
                 context=f"Dashboard ID: {dashboard_id}",
```

Comment:
just use exc=e here. don't need the `if isinstance(e, DeserializeError)` part

also the title should make it clear that this is an error with fetching from the Looker API, not necessarily in our code. "loading" is a bit too ambiguous